app/bundles/UserBundle/Event/PasswordStrengthValidateEvent.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\UserBundle\Event;
+
+use Symfony\Contracts\EventDispatcher\Event;
+
+class PasswordStrengthValidateEvent extends Event
+{
+    public function __construct(
+        private bool $isValid,
+        private string $password
+    ) {
+    }
+
+    public function isValid(): bool
+    {
+        return $this->isValid;
+    }
+
+    public function setIsValid(bool $isValid): PasswordStrengthValidateEvent
+    {
+        $this->isValid = $isValid;
+
+        return $this;
+    }
+
+    public function getPassword(): string
+    {
+        return $this->password;
+    }
+
+    public function setPassword(string $password): PasswordStrengthValidateEvent
+    {
+        $this->password = $password;
+
+        return $this;
+    }
```

Comment:
What do you think about this?:
```suggestion
    public function __construct(
        public bool $isValid,
        public string $password
    ) {
    }
```
And then instead of getters, setters, we can update the properties directly.

Reply:
I consider using getters and setters as a good practice. Why should we update the properties directly in this case?

Reply:
Because why do we need the getters and setters with typed properties and params? More thoughts on that for example in https://carstenwindler.de/php/reduce-boilerplate-code-with-php-8/

Reply:
OK, I get it. I've made the changes you suggested.